src/Tinlake.ts

```diff
@@ -340,9 +340,12 @@ export class Tinlake {
   }
 
   initFee = async (fee: string) => {
-    const txHash = await executeAndRetry(this.contracts.admin.file, [fee, fee, this.ethConfig]);
-    console.log(`[Pile.file] txHash: ${txHash}`);
-    return waitAndReturnEvents(this.eth, txHash, this.contracts.admin.abi, this.transactionTimeout);
+    // not all deployments include the new version of admin contract. call for some deployments file driectly from pile to avoid regression
+    const adminHasInitFeeMethod = await hasMethod(this.eth,this.contracts.admin.address,"file(uint256,uint256)")
```

Comment:
In the contract it is `file(uint fee, uint speed)` so `uint` not `uint256`.

But I think `uint` is internally `uint256`.

If the signature is just the hash of the string then it might be different.

Did you test both cases?

Reply:
yes, I tested with uint first. Didn't work